plugins/orchestrator/src/components/WorkflowDefinitionsListComponent/WorkflowDefinitionsListComponent.tsx

```diff
@@ -72,17 +72,17 @@ export const WorkflowsTable = ({ items }: WorkflowsTableProps) => {
   const getInitialState = useMemo(() => {
     return items.map(item => {
       return {
-        id: item.definition.id,
-        name: item.definition.name ?? '',
+        id: item.definition?.id,
```

Comment:
Does it make sense `definition` being `undefined` here?
If so, maybe we need to filter them out before `map`.